modules/gapi/samples/privacy_masking_camera.cpp

```diff
@@ -0,0 +1,215 @@
+#include <chrono>
+#include <iostream>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/gapi.hpp>
+#include <opencv2/gapi/core.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/infer.hpp>
+#include <opencv2/gapi/render.hpp>
+#include <opencv2/gapi/infer/ie.hpp>
+#include <opencv2/gapi/cpu/gcpukernel.hpp>
+#include <opencv2/gapi/streaming/cap.hpp>
+#include <opencv2/highgui.hpp>
+
+const std::string about =
+    "This is an OpenCV-based version of Privacy Masking Camera example";
+const std::string keys =
+    "{ h help |   | print this help message }"
+    "{ input  |   | Path to an input video file }"
+    "{ platm  |   | IE vehicle/license plate detection model IR }"
+    "{ platw  |   | IE vehicle/license plate detection model weights }"
+    "{ platd  |   | IE vehicle/license plate detection model device }"
+    "{ facem  |   | IE face detection model IR }"
+    "{ facew  |   | IE face detection model weights }"
+    "{ faced  |   | IE face detection model device }"
+    "{ trad   |   | Run processing in traditional (non-pipelined) way }"
+    "{ noshow |   | Don't display UI (improves performance) }";
+
+namespace {
+struct Avg {
+    struct Elapsed {
+        explicit Elapsed(double ms) : ss(ms/1000.), mm(static_cast<int>(ss)/60) {}
+        const double ss;
+        const int    mm;
+    };
+
+    using MS = std::chrono::duration<double, std::ratio<1, 1000>>;
+    using TS = std::chrono::time_point<std::chrono::high_resolution_clock>;
+    TS started;
+
+    void    start() { started = now(); }
+    TS      now() const { return std::chrono::high_resolution_clock::now(); }
+    double  tick() const { return std::chrono::duration_cast<MS>(now() - started).count(); }
+    Elapsed elapsed() const { return Elapsed{tick()}; }
+    double  fps(std::size_t n) const { return static_cast<double>(n) / (tick() / 1000.); }
+};
+std::ostream& operator<<(std::ostream &os, const Avg::Elapsed &e) {
+    os << e.mm << ':' << (e.ss - 60*e.mm);
+    return os;
+}
+} // namespace
```

Comment:
Why not [TickMeter](https://docs.opencv.org/master/d9/d6f/classcv_1_1TickMeter.html)?

Reply:
Never mind we have a TickMeter :D Ok will port

Reply:
Done